api/src/utils/env-config.ts

```diff
@@ -0,0 +1,122 @@
+import { z } from 'zod';
+import { getLogger } from './logger';
+
+const defaultLog = getLogger('src/utils/env-config.ts');
+
+// Custom Zod string type for environment variables ie: '' or ' ' are invalid
+const ZodEnvString = z.string().trim().min(1, { message: 'Required' });
+
+// Schema for environment configuration
+export const EnvSchema = z.object({
+  // Environment
+  NODE_ENV: z.enum(['development', 'test', 'production']),
+  NODE_OPTIONS: ZodEnvString,
+  TZ: z.literal('America/Vancouver'),
+
+  // API server
+  API_HOST: ZodEnvString,
+  API_PORT: z.coerce.number(),
```

Comment:
Should some of these number fields include a `required` constraint (similar to the string ones)?

Reply:
I added a schema for numbers that is more appropriate. 
ie: min of 0.